From 839fe5fafe510e3692644e8f7082e8818a236de1 Mon Sep 17 00:00:00 2001
From: Andrey <Andrey.Grodzovsky@amd.com>
Date: Mon, 10 Aug 2015 12:39:45 -0400
Subject: [PATCH 380/401] amd/dal: DP monitor can't light up with the eDP
 panel.

Fixed by moving init of resource range container before it's usage in tm_init_features.

Reviewed-By: Jordan Lazare <Jordan.Lazare@amd.com>
Signed-off-by: Andrey Grodzovsky <Andrey.Grodzovsky@amd.com>
Signed-off-by: Sanjay R Mehta <sanju.mehta@amd.com>
---
 drivers/gpu/drm/amd/dal/topology/topology.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/topology/topology.c b/drivers/gpu/drm/amd/dal/topology/topology.c
index d08967e..9ac01e8 100644
--- a/drivers/gpu/drm/amd/dal/topology/topology.c
+++ b/drivers/gpu/drm/amd/dal/topology/topology.c
@@ -4094,10 +4094,10 @@ static enum tm_result tm_init_during_construct(struct topology_mgr *tm)
 
 		associate_link_services_with_display_paths(tm);
 
-		tm_init_features(tm);
-
 		dal_tmrm_set_resources_range_by_type(tm->tm_rm);
 
+		tm_init_features(tm);
+
 		rc = tm_update_internal_database(tm);
 		if (TM_RESULT_FAILURE == rc)
 			break;
-- 
1.9.1

